content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,108 @@
+
+You're right to be careful, because `dvc gc --cloud` can be dangerous in the
+wrong hands- it'll remove any unused files in your S3 remote (for more info,
+[see our docs](https://dvc.org/doc/command-reference/gc#gc)). To prevent users
+from having this power, setting your S3 bucket policy to block object deletions
+should do the trick (check out this
+[relevant Stack Overflow question](https://stackoverflow.com/questions/49693706/disable-delete-option-for-s3-objects-in-aws)).
+
+### Q: [My team is interested in DVC, and we have all of our data in S3 remote storage. Do we need to install a centralised enterprise version of DVC on a dedicated server? And do we have to also have a GitHub repository?](https://discordapp.com/channels/485586884165107732/563406153334128681/692524884701478992)
+
+There's no need for a DVC server. Our remote storage works on top of S3 by
+default, with no additional infrastructure required. As for GitHub (or
+BitBucket, or GitLab, etc.), this is only needed if you're interested in sharing
+your project with others over that channel. We _like_ sharing projects on
+GitHub, but you don't have to. Any Git repository, even a local one, will do.
+
+So a "minimal" DVC project for you might consist of a local workspace with Git
+enabled (which you _do_ need), a local Git repository, and your S3 remote
+storage. Check out our
+[use cases](https://dvc.org/doc/use-cases/versioning-data-and-model-files) to
+see some examples of infrastructure and workflow for teams.
+
+### Q: [Could there be any issues with concurrent `dvc push`-es to the same remote?](https://discordapp.com/channels/485586884165107732/563406153334128681/680053750320332800)
+
+There are a few ways for concurrency to occur: multiple jobs running in parallel
+on the same machine, or different users on different machines. But in any case,
+the answer is the same: there's nothing to worry about! When pushing a file to a
+DVC remote, all operations are non-destructive and atomic.
+
+### Q: [How do I only download part of my remote repository? For example, I only need the final output of my pipeline, not the raw data or intermediate steps.](https://discordapp.com/channels/485586884165107732/485596304961962003/696751934777852004)
+
+We support granular operations on DVC project repositories! Say your project's
+DVC remote contains several `.dvc` files corresponding to different stages of
+your pipeline: `0_process_data.dvc`, `1_split_test_train.dvc`, and
+`2_train_model.dvc`. If you're only interested in the files output by the final
+stage of the pipeline (`2_train_model.dvc`), you can run:
+
+```dvc
+$ dvc pull process_data_stage.dvc
+```
+
+You can also use `dvc pull` at the level of individual files. For example, if
+you wanted to pull `model.pkl`, a trained model in remote DVC storage with a
+corresponding `model.pkl.dvc` file in your local workspace, you would simply run
```

Comment:
so, if there is no single corresponding file (one stage produces 10 files) you can just run `dvc pull model.pkl`

Reply:
I think, since we mention that we can do `dvc pull something.dvc ` previously it makes sense in the second part to keep `dvc pull model.pkl`